01-Authorization/build.gradle

```diff
@@ -5,11 +5,10 @@ buildscript {
 }
 
 plugins {
-    id 'org.springframework.boot' version '1.5.21.RELEASE'
+    id 'org.springframework.boot' version '1.5.22.RELEASE'
 }
 
-ext['jackson.version'] = '2.9.9'
-ext['logback.version'] = '1.2.3'
+apply plugin: 'io.spring.dependency-management'
```

Comment:
using the `apply` instead of including in the `plugins` DSL, so it uses the version specified in the `org.springframework.boot` plugin.